### ISSUE TITLE
Accessibility Test Fails to Execute (Google Lighthouse)

### ISSUE THREAD

**Issue by Quartzion** [Quartzion/qts-v1.3.0#55]
**Actual** When a user executes a google lighthouse report, the accessibility test will not render results.
**Expected** When a user executes a google lighthouse report all tests execute and return results.

**Technical Issue**
- [ ] the accessibility fails due to the paypal sdk (note - warning flags returned when the donate modal is expanded) 
     - [ ] this issue can be resolved by enhancing the lazy loading with the following strategy

**Resolution Strategy**
1. eliminate paypal provider wrapper from app.jsx
2. apply to the individual nested elements 

```ruby
import { PayPalScriptProvider } from "@paypal/react-paypal-js";
import { isProd, getApiBaseUrl, getPayPalClientId } from '../../utils/env';
const QtsPayPal = React.lazy(() => import("../QtsPayPal"));

        <Suspense fallback={<div>Loading payment options…</div>}>
          <PayPalScriptProvider options={{ "client-id": VITE_PAYPAL_APP_CLIENT }}>
            <QtsPayPal />
          </PayPalScriptProvider>
        </Suspense>
```
